lib/mayaUsd/render/vp2RenderDelegate/proxyRenderDelegate.h

```diff
@@ -191,6 +191,21 @@ class ProxyRenderDelegate
 
     MAYAUSD_CORE_PUBLIC
     void DisplayLayerDirty(MFnDisplayLayer& displayLayer);
+
+    MAYAUSD_CORE_PUBLIC
+    void DisplayLayerPathChanged(const Ufe::Path& oldPath, const Ufe::Path& newPath);
+
+    MAYAUSD_CORE_PUBLIC
+    void AddDisplayLayerToCache(MObject& displayLayerObj);
+
+    MAYAUSD_CORE_PUBLIC
+    void UpdateProxyShapeDisplayLayers();
+
+    MAYAUSD_CORE_PUBLIC
+    MObject GetDisplayLayer(const SdfPath& path);
+
+    MAYAUSD_CORE_PUBLIC
+    const MObjectArray& GetProxyShapeDisplayLayers() const { return _usdStageDisplayLayers; }
```

Comment:
This class in general really lacks comments about its design and functions... but maybe for another day.